pkgs/tools/misc/hdf4/default.nix

```diff
@@ -12,6 +12,8 @@
 , szip
 , javaSupport ? false
 , jdk
+, fortranSupport ? false
```

Comment:
This can actually be `true` per default. The problem is not the Fortran support but the NetCDF support, which breaks `gdal`.

Reply:
Ah you are right, I've messed it up. Is fixed.

Reply:
OK, defaults to false again. I am surprised how fragile this all is ^^